ws4redis/websocket.py

```diff
@@ -198,14 +198,14 @@ def receive(self):
             raise WebSocketError("Connection is already closed")
         try:
             return self.read_message()
-        except UnicodeError:
-            logger.info('websocket.receive: UnicodeError')
+        except UnicodeError as e:
+            logger.info(u'websocket.receive: UnicodeError {}'.format(e))
             self.close(1007)
-        except WebSocketError:
-            logger.info('websocket.receive: WebSocketError')
+        except WebSocketError as e:
+            logger.info(u'websocket.receive: WebSocketError {}'.format(e))
             self.close(1002)
         except Exception as e:
-            logger.info('websocket.receive: Unknown error %s', e)
+            logger.info(u'websocket.receive: Unknown error {}'.format(e))
```

Comment:
Just a note. I pilfered this file, therefore the old string formatting was used. Is this file now Py3 safe?